src/main/java/org/alfresco/repo/solr/NodeParameters.java

```diff
@@ -54,10 +54,11 @@
     private Set<QName> excludeNodeTypes;
     
     private Set<QName> includeAspects;
-    private Set<QName> excludeAspects;
-    
-    private String shardProperty;
+    private Set<QName> excludeAspects;
     
+    private String shardProperty;
+    private String coreName;
+

```

Comment:
This change only includes "coreName" field, but it changes other lines... Again the spaces-tab fighting?